CIP-0021/README.md

```diff
@@ -71,8 +72,16 @@ Unless mentioned otherwise in this CIP, optional empty lists and maps must not b
 
 **Outputs**
 
+A new, "post Alonzo", output format has been introduced in the Babbage era which uses a map instead of an array to store the output data. For now, both the "legacy" (array) and "post Alonzo" (map) output formats are supported by HW wallets but we encourage everyone to migrate to the "post Alonzo" format as support for the "legacy" output format might be removed in the future. A single transaction can include outputs with both formats.
+
+_Legacy outputs_
+
 Outputs containing no multi-asset tokens must be serialized as a simple tuple, i.e. `[address, coin, ?datum_hash]` instead of `[address, [coin, {}], ?datum_hash]`. In addition, including `datum_hash` is only allowed if the payment part of the output's `address` is a script hash.
 
+_Post Alonzo outputs_
+
+No additional restrictions apply.
```

Comment:
Ledger expects non-empty inline datum and ref. script. Maybe it's not necessary, but it seems impossible for them to be empty anyway:
https://input-output-rnd.slack.com/archives/G010N9UFDNH/p1654246117932079?thread_ts=1654160314.762289&cid=G010N9UFDNH

Reply:
I tried mentioning it -> https://github.com/vacuumlabs/CIPs/pull/8/commits/aa212f76214f73c466b85879fdb5d3891499b3dc#diff-5f939a65a33a4da4a1743d1bcafda7134e786496e39b9494eaf287da04ea2187R83.